cmd/manager/main.go

```diff
@@ -349,7 +352,15 @@ func main() {
 
 	setupLog.Info("Starting manager")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
+		analytics.SendAnonymousInfo(ctx, controllerConfigManager.Client, "kusk", fmt.Sprintf("kusk-gateway manager bootstrapping failed with error %s", err.Error()))
 		setupLog.Error(err, "Problem running manager")
 		os.Exit(1)
 	}
 }
+func heartBeat(ctx context.Context, client client.Client) {
+	c := cron.New()
+	c.AddFunc("@daily", func() {
+		analytics.SendAnonymousInfo(ctx, client, "kusk-heartbeat", "kusk-gateway daily heartbeat")
```

Comment:
Is this meant to be `"kusk"`? or `"kusk-heartbeat"`? I see `"kusk"` being used in most places.

Reply:
it was intentional to differentiate events.
Segment will create two separate datatables in PG one each event. That way it will be way simpler to track what we need to